### DISCUSSION TITLE
Groups should be merged into eachother instead of only spawning from the current one

### DISCUSSION THREAD

**Discussion by txtyash** [hyprwm/Hyprland#2609] [General]
In the dwindle layout creating a group requires spawning new windows which get added to the group but already open windows should also have the ability to be added in a pre-existing group. This can be done by adding windows to the group depending on the keybindings.

Let's say i have bindings `mod+shift+l` it would merge window on the right of the focused window into the group, similarly `mod+shift+h` would merge the window on the left into the group and similar... This would work better than just spawned group windows.

**Comment by MightyPlaza** [comment]
You can already do this with the mouse

**Comment by MightyPlaza** [comment]
There are also dispatchers moveintogroup and moveoutofgroup, which I'm guessing does what you asked

**Reply by txtyash** [reply]
Thanks! I should've looked into the dispatcher list. I was looking at the `dwindle` section.
